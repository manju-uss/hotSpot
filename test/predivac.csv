epitope,coverage,hla_number,start,end,hlas
ITGLHPTQA,17.19,2,15,23,HLA-DRB1*01:01;HLA-DRB1*01:03
YRRLISMMG,48.16,6,51,59,HLA-DRB1*01:01;HLA-DRB1*01:03;HLA-DRB1*04:01;HLA-DRB1*04:05;HLA-DRB1*10:01;HLA-DRB1*11:01
FVKIGPERT,26.04,4,198,206,HLA-DRB1*01:01;HLA-DRB1*01:03;HLA-DRB1*13:02;HLA-DRB1*14:02
WTIEYPIIG,24.31,3,292,300,HLA-DRB1*01:01;HLA-DRB1*01:03;HLA-DRB1*08:01
YVPLKSATC,19.0,3,465,473,HLA-DRB1*01:01;HLA-DRB1*01:03;HLA-DRB1*09:01
YNMMISAGF,29.44,5,498,506,HLA-DRB1*01:01;HLA-DRB1*01:03;HLA-DRB1*12:01;HLA-DRB1*13:02;HLA-DRB1*14:02
LFKDCSKVI,31.11,2,7,15,HLA-DRB1*03:01;HLA-DRB1*14:01
LSVDTKFKT,26.04,1,27,35,HLA-DRB1*03:01
IPKDMTYRR,32.76,3,45,53,HLA-DRB1*03:01;HLA-DRB1*04:02;HLA-DRB1*04:04
IGFDVEGCH,31.11,2,87,95,HLA-DRB1*03:01;HLA-DRB1*14:01
FMIDVQQWG,31.11,2,240,248,HLA-DRB1*03:01;HLA-DRB1*14:01
VLHDIGNPK,32.76,3,328,336,HLA-DRB1*03:01;HLA-DRB1*04:02;HLA-DRB1*04:04
FSTGVNLVA,40.71,6,111,119,HLA-DRB1*04:01;HLA-DRB1*04:02;HLA-DRB1*04:04;HLA-DRB1*04:05;HLA-DRB1*11:01;HLA-DRB1*11:02
YVDTPNNTD,53.76,6,124,132,HLA-DRB1*04:01;HLA-DRB1*04:03;HLA-DRB1*04:05;HLA-DRB1*07:01;HLA-DRB1*08:01;HLA-DRB1*10:01
WGFTGNLQS,40.71,6,247,255,HLA-DRB1*04:01;HLA-DRB1*04:02;HLA-DRB1*04:03;HLA-DRB1*04:04;HLA-DRB1*04:05;HLA-DRB1*08:01
FYYSDSPCE,48.16,5,445,453,HLA-DRB1*04:01;HLA-DRB1*04:03;HLA-DRB1*04:05;HLA-DRB1*07:01;HLA-DRB1*09:01
YYSDSPCES,39.16,5,446,454,HLA-DRB1*04:01;HLA-DRB1*04:05;HLA-DRB1*08:01;HLA-DRB1*10:01;HLA-DRB1*14:01
ITREEAIRH,7.84,2,74,82,HLA-DRB1*04:02;HLA-DRB1*04:04
LHDIGNPKA,51.0,8,329,337,HLA-DRB1*04:02;HLA-DRB1*04:03;HLA-DRB1*04:04;HLA-DRB1*09:01;HLA-DRB1*11:01;HLA-DRB1*11:02;HLA-DRB1*12:01;HLA-DRB1*15:01
LHPTQAPTH,3.96,2,18,26,HLA-DRB1*04:03;HLA-DRB1*09:01
LWNTFTRLQ,1.99,1,519,527,HLA-DRB1*04:03
MFITREEAI,22.56,1,72,80,HLA-DRB1*07:01
FDYVYNPFM,67.51,6,233,241,HLA-DRB1*07:01;HLA-DRB1*09:01;HLA-DRB1*13:01;HLA-DRB1*13:02;HLA-DRB1*14:02;HLA-DRB1*15:01
FVKRVDWTI,22.56,1,286,294,HLA-DRB1*07:01
FYDAQPCSD,24.31,2,350,358,HLA-DRB1*07:01;HLA-DRB1*10:01
FKDCSKVIT,7.84,1,8,16,HLA-DRB1*08:01
IGDELKINA,9.75,2,299,307,HLA-DRB1*08:01;HLA-DRB1*11:02
FYSYATHSD,1.99,1,367,375,HLA-DRB1*09:01
FKHLIPLMY,15.36,4,146,154,HLA-DRB1*10:01;HLA-DRB1*12:01;HLA-DRB1*13:02;HLA-DRB1*14:02
YCQVHGNAH,11.64,3,260,268,HLA-DRB1*10:01;HLA-DRB1*13:02;HLA-DRB1*14:02
VTGLFKDCS,24.31,3,4,12,HLA-DRB1*11:01;HLA-DRB1*11:02;HLA-DRB1*13:01
YACWHHSIG,7.84,1,224,232,HLA-DRB1*11:01
VLSNLNLPG,32.76,5,405,413,HLA-DRB1*11:01;HLA-DRB1*11:02;HLA-DRB1*12:01;HLA-DRB1*13:01;HLA-DRB1*14:01
IPGIPKDMT,17.19,2,42,50,HLA-DRB1*11:02;HLA-DRB1*13:01
VGTNLPLQL,3.96,1,101,109,HLA-DRB1*12:01
IVQMLSDTL,19.0,2,166,174,HLA-DRB1*12:01;HLA-DRB1*13:01
LFWNCNVDR,15.36,1,383,391,HLA-DRB1*13:01
YPIIGDELK,9.75,2,296,304,HLA-DRB1*13:02;HLA-DRB1*14:02
VSAKPPPGD,5.91,1,136,144,HLA-DRB1*14:01
VNGYPNMFI,31.11,1,66,74,HLA-DRB1*15:01
VASCDAIMT,31.11,1,269,277,HLA-DRB1*15:01
VCLFWNCNV,31.11,1,381,389,HLA-DRB1*15:01
VDRYPANSI,31.11,1,389,397,HLA-DRB1*15:01
