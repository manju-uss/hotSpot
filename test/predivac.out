,coverage,alleles
ITGLHPTQA,17.1900,"HLA-DRB1*01:01,HLA-DRB1*01:03"
YRRLISMMG,48.1600,"HLA-DRB1*01:01,HLA-DRB1*01:03,HLA-DRB1*04:01,HLA-DRB1*04:05,HLA-DRB1*10:01,HLA-DRB1*11:01"
FVKIGPERT,26.0400,"HLA-DRB1*01:01,HLA-DRB1*01:03,HLA-DRB1*13:02,HLA-DRB1*14:02"
WTIEYPIIG,24.3100,"HLA-DRB1*01:01,HLA-DRB1*01:03,HLA-DRB1*08:01"
YVPLKSATC,19.0000,"HLA-DRB1*01:01,HLA-DRB1*01:03,HLA-DRB1*09:01"
YNMMISAGF,29.4400,"HLA-DRB1*01:01,HLA-DRB1*01:03,HLA-DRB1*12:01,HLA-DRB1*13:02,HLA-DRB1*14:02"
LFKDCSKVI,31.1100,"HLA-DRB1*03:01,HLA-DRB1*14:01"
LSVDTKFKT,26.0400,HLA-DRB1*03:01
IPKDMTYRR,32.7600,"HLA-DRB1*03:01,HLA-DRB1*04:02,HLA-DRB1*04:04"
IGFDVEGCH,31.1100,"HLA-DRB1*03:01,HLA-DRB1*14:01"
FMIDVQQWG,31.1100,"HLA-DRB1*03:01,HLA-DRB1*14:01"
VLHDIGNPK,32.7600,"HLA-DRB1*03:01,HLA-DRB1*04:02,HLA-DRB1*04:04"
FSTGVNLVA,40.7100,"HLA-DRB1*04:01,HLA-DRB1*04:02,HLA-DRB1*04:04,HLA-DRB1*04:05,HLA-DRB1*11:01,HLA-DRB1*11:02"
YVDTPNNTD,53.7600,"HLA-DRB1*04:01,HLA-DRB1*04:03,HLA-DRB1*04:05,HLA-DRB1*07:01,HLA-DRB1*08:01,HLA-DRB1*10:01"
WGFTGNLQS,40.7100,"HLA-DRB1*04:01,HLA-DRB1*04:02,HLA-DRB1*04:03,HLA-DRB1*04:04,HLA-DRB1*04:05,HLA-DRB1*08:01"
FYYSDSPCE,48.1600,"HLA-DRB1*04:01,HLA-DRB1*04:03,HLA-DRB1*04:05,HLA-DRB1*07:01,HLA-DRB1*09:01"
YYSDSPCES,39.1600,"HLA-DRB1*04:01,HLA-DRB1*04:05,HLA-DRB1*08:01,HLA-DRB1*10:01,HLA-DRB1*14:01"
ITREEAIRH,7.8400,"HLA-DRB1*04:02,HLA-DRB1*04:04"
LHDIGNPKA,51.0000,"HLA-DRB1*04:02,HLA-DRB1*04:03,HLA-DRB1*04:04,HLA-DRB1*09:01,HLA-DRB1*11:01,HLA-DRB1*11:02,HLA-DRB1*12:01,HLA-DRB1*15:01"
LHPTQAPTH,3.9600,"HLA-DRB1*04:03,HLA-DRB1*09:01"
LWNTFTRLQ,1.9900,HLA-DRB1*04:03
MFITREEAI,22.5600,HLA-DRB1*07:01
FDYVYNPFM,67.5100,"HLA-DRB1*07:01,HLA-DRB1*09:01,HLA-DRB1*13:01,HLA-DRB1*13:02,HLA-DRB1*14:02,HLA-DRB1*15:01"
FVKRVDWTI,22.5600,HLA-DRB1*07:01
FYDAQPCSD,24.3100,"HLA-DRB1*07:01,HLA-DRB1*10:01"
FKDCSKVIT,7.8400,HLA-DRB1*08:01
IGDELKINA,9.7500,"HLA-DRB1*08:01,HLA-DRB1*11:02"
FYSYATHSD,1.9900,HLA-DRB1*09:01
FKHLIPLMY,15.3600,"HLA-DRB1*10:01,HLA-DRB1*12:01,HLA-DRB1*13:02,HLA-DRB1*14:02"
YCQVHGNAH,11.6400,"HLA-DRB1*10:01,HLA-DRB1*13:02,HLA-DRB1*14:02"
VTGLFKDCS,24.3100,"HLA-DRB1*11:01,HLA-DRB1*11:02,HLA-DRB1*13:01"
YACWHHSIG,7.8400,HLA-DRB1*11:01
VLSNLNLPG,32.7600,"HLA-DRB1*11:01,HLA-DRB1*11:02,HLA-DRB1*12:01,HLA-DRB1*13:01,HLA-DRB1*14:01"
IPGIPKDMT,17.1900,"HLA-DRB1*11:02,HLA-DRB1*13:01"
VGTNLPLQL,3.9600,HLA-DRB1*12:01
IVQMLSDTL,19.0000,"HLA-DRB1*12:01,HLA-DRB1*13:01"
LFWNCNVDR,15.3600,HLA-DRB1*13:01
YPIIGDELK,9.7500,"HLA-DRB1*13:02,HLA-DRB1*14:02"
VSAKPPPGD,5.9100,HLA-DRB1*14:01
VNGYPNMFI,31.1100,HLA-DRB1*15:01
VASCDAIMT,31.1100,HLA-DRB1*15:01
VCLFWNCNV,31.1100,HLA-DRB1*15:01
VDRYPANSI,31.1100,HLA-DRB1*15:01
